.gitignore

```diff
@@ -0,0 +1,6 @@
+
+# Mac stuff
+*.DS_Store
+
+# R stuff
+uptasticsearch.Rcheck
```

Comment:
Just pushed some changes. I used this for inspiration: https://gist.github.com/AnthonyEbert/b4b90dc40f62a41365627019bbdd057a